testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/authentication/FlowTest.java

```diff
@@ -474,7 +474,7 @@ public boolean test(AuthenticationFlowRepresentation rep) {
         } catch (InternalServerErrorException isee) {
             try (Response response = isee.getResponse()) {
                 assertEquals(500, response.getStatus());
-                assertFalse(StreamUtil.readString((InputStream) response.getEntity(), Charset.forName("UTF-8")).toLowerCase().contains("exception"));
+                assertEquals(0, response.getLength());
```

Comment:
Why is this changed?

Reply:
It's revert from the https://github.com/keycloak/keycloak/pull/15193/files#diff-40281afe608b4070873d6915e1e4070f0f091611294e6450fbd50863a6154e86L468.